oci/manager.go

```diff
@@ -123,6 +130,19 @@ func (manager *ociManager) Catalog(ctx context.Context, qopts ...packmanager.Que
 	query := packmanager.NewQuery(qopts...)
 	qname := query.Name()
 	qversion := query.Version()
+	platform := &v1.Platform{}
+
+	if query.Architecture() != "" {
+		platform.Architecture = query.Architecture()
+	}
+
+	if query.OS() != "" {
+		platform.OS = query.OS()
+	}
```

Comment:
We retrofit platform into the OS, so we should keep our naming conventions. Please rename the `OS` attribute to `Platform` of the query.